src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebefore/Example1.java

```diff
@@ -12,7 +12,7 @@ class Example1 {
   int foo = 5;
   void example() {
     // xdoc section -- start
-    foo ++; // violation ''\+\+' is preceded with whitespace'
```

Comment:
The example is understandable without mentioning the pluses.

Reply:
It is to avoid escaping that will looks weird in html rendering from xdoc content.